### DISCUSSION TITLE
update on MacBook m1 ableton live 10.3.1 (rosetta)

### DISCUSSION THREAD

**Discussion by andri3Andreas** [apohl79/audiogridder#370] [Help] [answered]
previous version worked. now I get this (to verify Logic Pro X works fine, but is arm64 optimized) 
<img width="1680" alt="Screenshot 2021-02-08 at 11 35 25" src="https://user-images.githubusercontent.com/78731075/107209456-1f72de80-6a03-11eb-9e91-dfe36c6d7dd5.png">

**Comment by andri3Andreas** [comment]
is there a way to flush the preset cache ?! previous saved Ableton projects with Audiograbber plugins remain ok and openable

**Comment by andri3Andreas** [comment, marked as answer]
<img width="1680" alt="Screenshot 2021-02-09 at 12 25 46" src="https://user-images.githubusercontent.com/78731075/107365963-d80d5080-6add-11eb-9254-e34e594f4796.png">

this is what the error message reads for new instances of audio grabber after updating

**Reply by apohl79** [reply]
Are you running the same version on the server and plugin side?

**Reply by andri3Andreas** [reply]
well I had to fiddle back n forth to even make it work in Ableton Live, Since it is using rosetta (M1 air) to translate from arm64 , and the parallels desktop uses an arm64 win 10 beta version.

So logic audio x pro (m1 optimised) works like a charm with the parallels dekstop win 10.

Ableton worked until I updated

I am actually not sure ifAbleton uses the arm64 plugin version of Audiogridder or the intel

**Reply by apohl79** [reply]
> I am actually not sure ifAbleton uses the arm64 plugin version of Audiogridder or the intel

As the binary versions are not bundled together in a universal binary, you can either install the intel or the arm version.

**Reply by andri3Andreas** [reply]
I know and I've installed both, I meant I am not sure which one Ableton uses

(I found this out)
* it says 1.1 beta, After   I have copied  the latest installment from ur website (macos-x86_64 Binaries)
it still shows 1.1 beta

if I remove them, the arm64 shows in the plugin menu of Ableton but doesn't open and same in logic X, which is M1 optimised

So both apps use the latest 10.8 + version (1.1 beta), But in Ableton no consistent fx menu (after rebuilding caches on the server side)

* Update, after manual removing and adding the intel 10.8 + binary's again Logi X also has the menu issues : (

**Reply by andri3Andreas** [reply]
<img width="422" alt="Screenshot 2021-02-10 at 16 27 25" src="https://user-images.githubusercontent.com/78731075/107532247-f8144100-6bbd-11eb-8708-86c1c78d9fc2.png">


Im afraid to reinstall everything (it took some filling to make it actually work) .

to sum it up: previously saved states are represented well. adding new plugins in the audiogridder plugin gives the scrambled menu and this message)

**Reply by apohl79** [reply]
The problem is, that you are using the beta11 plugin and the 1.1.0 server. Can you upgrade the plugin to 1.1.0? The error should go away than.

**Reply by andri3Andreas** [reply]
So since I m on an M1 system with two different apps (ableton Live 10 rosetta and Logic Pro X m1) I installed both in thisorder

1. AudioGridderPlugin_1.1.0_macOS-arm64

Result: audiogridder shows up and works in Logic X BUT NOT in ableton live


2. I install AudioGridderPlugin_1.1.0_macOS-x86_64

Result: audiogridder shows up and works in Ableton BUT NOT in  Logic Pro X

3. Installing AudioGridderPlugin_1.1.0_macOS-arm64 BUT this time ONLY  AU

Result:  audiogridder shows up and works in Logic X  (AUComponent) And ALSO in ableton live (VST only)

So because of the M1 system there is still the  M1 optimised vs Rosetta issue

But I can use them in both programs! and menu is Fixed  

thank u for the effort! and love the concept and stability of the app.
